src/aggregator/client/writer.go

```diff
@@ -288,7 +331,7 @@ func (w *writer) encodeUntimedWithLock(
 	default:
 	}
 
-	return errUnrecognizedMetricType
+	return ErrUnrecognizedMetricType
```

Comment:
Why are these public now? Seems unnecessary?

Reply:
Let's make that one error public, but leave the rest private. Generally speaking best to minimize sentinel errors as part of the package API.